### ISSUE TITLE
Issue with the Rewired package and FSR 1.6

### ISSUE THREAD

**Issue by nergethic** [handzlikchris/FastScriptReload#122]
I'm currently using Unity 2022.3.16f, and FSR 1.5 is working flawlessly. However, when I upgrade the package to version 1.6 or use the latest development version (5b61e98), I encounter the following errors:
"Rewired: Rewired is not initialized. You must have an active and enabled Rewired Input Manager in the scene before calling any part of the Rewired API"

**Comment by handzlikchris** [comment]
I don't have rewired and won't be able to check. Nothing initialization wise should change from 1.5 to 1.6 as far as I remember.

Are you 100% sure changing versions (without changing anything else like domain-reload) gets it working?

**Comment by nergethic** [comment]
Yes, I tested it multiple times. I was deleting the Library folder before updating the version and was doing the same live code changes. I forgot to add that this error happens only after I change the code in play mode.
